mail-toaster.sh

```diff
@@ -535,6 +539,7 @@ has_data_fs()
 		postgres ) return 0;;
 		haproxy )  return 0;;
 		minecraft ) return 0;;
+		php7 )     return 0;;
```

Comment:
That's right no use in the data partition (probably added it first time)

I do share data between the PHP7 and nginx, that works perfectly.

I do have this jail.conf

`mount += "/jails/nginx/sites $path/sites nullfs rw 0 0";
`
In the folder /sites all websites  are located since PHP need access to the files